src/bidiMapper/domains/context/context.ts

```diff
@@ -108,6 +110,108 @@ export class Context {
     });
   }
 
+  private _initializeLogEntryAddedEventListener() {
+    this._cdpClient.Runtime.on('consoleAPICalled', async (params) => {
+      const args = await Promise.all(
+        params.args.map(async (arg) => {
+          return (await this._serializeCdpObject(arg)) as Log.RemoteValue;
+        })
+      );
+
+      const stackFrames = params.stackTrace?.callFrames.map((callFrame) => {
+        return {
+          columnNumber: callFrame.columnNumber,
+          functionName: callFrame.functionName,
+          lineNumber: callFrame.lineNumber,
+          url: callFrame.url,
+        };
+      });
+      const stackTrace = stackFrames ? { callFrames: stackFrames } : undefined;
+
+      const getLogLevel = (consoleAPIType: string) => {
+        if (consoleAPIType in ['error', 'assert']) {
+          return 'error';
+        }
+        if (consoleAPIType in ['debug', 'trace']) {
+          return 'debug';
+        }
+        if (consoleAPIType in ['warning', 'warn']) {
+          return 'warning';
+        }
+        return 'info';
+      };
+
+      const specifiers = ['%s', '%d', '%i', '%f'];
```

Comment:
What about `%o`, `%O`, `%c` specified in [2.2.1. Summary of formatting specifiers](https://console.spec.whatwg.org/#formatting-specifiers)?

Reply:
Implemented %o, %O and %c in formatter.